fork/mklishevych/actions.js

```diff
@@ -0,0 +1,39 @@
+export const successAction = (text) => {
```

Comment:
is there really big difference between actions type?

Reply:
no, just level :) so yes, further refactoring might be extracting this to a function `notificationAction(text, level)` and after that `successAction` and `errorAction` would look like this:
```javascript
export const successAction(text) => notificationAction(text, 'success');
export const errorAction(text) => notificationAction(text, 'error')
```

Reply:
that looks better and know small question what is success and error string for?

Reply:
I don't have an answer on that :) I made my code backward compatible. I don't know how action payload is actually used in the app. Maybe the app behaves differently depending on the `level` when the action of type `ADD_NOTIFICATION_MIDDLEWARE ` is dispatched.